src/test/scala/chiselTests/VectorPacketIO.scala

```diff
@@ -0,0 +1,57 @@
+// See LICENSE for license details.
+
+package chiselTests
+
+import Chisel._
+import Chisel.testers.BasicTester
+
+/**
+  * This test used to fail when assignment statements were
+  * contained in DeqIO and EnqIO constructors.
+  * The symptom is creation of a firrtl file
+  * with missing declarations, the problem is exposed by
+  * the creation of the val outs in VectorPacketIO
+  * NOTE: The problem does not exist now because the initialization
+  * code has been removed from DeqIO and EnqIO
+  */
+class Packet extends Bundle {
```

Comment:
The failure mode does not occur until the firrtl stage, so I am not sure how I would implement this as a unittest.